### DISCUSSION TITLE
AzureRM PowerShell modules are retiring on 29 February 2024

### DISCUSSION THREAD

**Discussion by mbraekman** [azure-deprecation/dashboard#119] [Deprecation]
Received a mail-notification stating that AzureRM PowerShell modules are retiring on 29 February 2024.

----
**_You're receiving this email because you use AzureRM PowerShell modules._** 

Because Az PowerShell modules now have all the capabilities of AzureRM PowerShell modules and more, we'll retire AzureRM PowerShell modules on 29 February 2024. 

We encourage you to start using Az PowerShell modules sooner to take advantage of these improvements: 
•	Greater security with token cache encryption and improved authentication. 
•	Availability in Azure Cloud Shell and on Linux and macOS. 
•	Support for all Azure services. 

**Required action** 
To avoid service interruptions, [update your scripts](http://aka.ms/azpsmigrate) that use AzureRM PowerShell modules to use Az PowerShell modules by 29 February 2024. To automatically update your scripts, follow the [quickstart guide](https://docs.microsoft.com/powershell/azure/quickstart-migrate-azurerm-to-az-automatically). 

**More information** 
If you have questions, get answers from Microsoft and community experts on [GitHub](https://github.com/Azure/azure-powershell/issues). 
If you have a support plan and need technical help, please create a [support request](https://portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportrequest): 
1.	Under _Issue type_, select **Technical**. 
2.	Under _Subscription_, select your subscription. 
3.	Under _Service_, select **My services**, then select the service related to PowerShell module that you're using. 
4.	If _Resource_ is required, select your resource. 
5.	Under _Summary_, type a description of your issue. 
6.	Under _Problem type_ and _Problem subtype_, select the options related to the PowerShell cmdlet that you're using. 

----

**Comment by tomkerkhove** [comment]
So it begins... Thanks for reporting!

**Reply by tomkerkhove** [reply]
Published - Thanks a ton!
https://github.com/azure-deprecation/dashboard/issues/120
